### DISCUSSION TITLE
Logs

### DISCUSSION THREAD

**Discussion by ptitmouton** [lotta-schule/web#284] [Ideen]
Ein Log was so am System passiert

## Zusammenfassung aller Ideen
Im Log sollten folgende Einträge erscheinen (gerne weiter Vorschläge hinzufügen)

Jeder Log-Eintrag soll mindestens folgende Daten vorweisen:
- Art
- Zeitstempel
- ausführender Nutzer
(bsp.: Beitrag gelöscht von Max Mustermann am 1.1.1900 um 12:37)

Zusätzlich sollten weitere Informationen verfügbar sein, die sich auf die spezifische Aktion beziehen. In der folgenden Liste in Klammern:

### Beiträge
- Beitrag bearbeitet (inkl. Titel + und Veränderungen)
- Beitrag gelöscht (inkl. Titel)

### Dateien und Medien
- Dateiumwandlung angestossen (inkl. Format und Länge) // Das können wir dann auch für genauere Abrechnung nutzen

### Nutzer
- Nutzer hat sich registriert
- Nutzer gelöscht (inkl. Name und Email)
- Nutzer Rechte zugewiesen (inkl. Name und Email)

### Gruppen
- Gruppe bearbeitet (inkl. Name und Veränderungen)
- Gruppe gelöscht (inkl. Name und Veränderungen)

### Kategorien
- Kategorie erstellt (inkl. Name)
- Kategorie bearbeitet (inkl. Name und Veränderungen)
- Kategorie gelöscht (inkl. Name)

### Marginalen
- Marginale bearbeitet (inkl. Name und Veränderungen)
- Marginale gelöscht (inkl. Name)

Das wären denke ich die wichtigsten Aktionen, um folgendes zu erreichen:
- Administrationsfehler schnell erkennen und korrigieren zu können (zB der Fall als plötzlich Beiträge für alle sichtbar waren, hiermit ließe sich überprüfen woran das liegen könnte)
- Nutzer überwachen, dass sie kein Unfug treiben (hat jemand einfach Beiträge gelöscht, nach Freigabe böswillig bearbeitet, ...)
- Bessere Transparenz der Kosten für den Nutzer erreichen (Wieso haben wir so viel Medienumwandlungen?)
- Unsere eigene Abrechnung genauer machen und die ein oder andere Mediumumwandlung in Rechnung stellen, die eventuell bisher unabgerechnet blieb

Je nachdem was uns davon wichtig ist könnten wir überlegen, Sachen wegzulassen oder hinzuzufügen, bzw. zu priorisieren. Denke nicht dass wir alles in einem Rutsch machen werden.

**Comment by techotline** [comment]
Für Änderungen an Beiträgen

**Comment by wie0815** [comment]
Es wäre schön, wenn man folgendes sehen kann:
vollständiger Name des Nutzers | Datum (Speichern!) | Zeit (Speichern!) | Link zum Beitrag
